pygame/math.py

```diff
@@ -0,0 +1,1575 @@
+# pygame_cffi - a cffi implementation of the pygame library
+# Copyright (C) 2016  Anton Joubert
+# Copyright (C) 2016  Simon Cross
+# Copyright (C) 2016  Neil Muller
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the Free
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA  02110-1301  USA
+
+""" Math module """
+
+from __future__ import absolute_import, division
+import math
+from numbers import Number
+
+from pygame.compat import string_types, pow_compat
+
+VECTOR_EPSILON = 1e-6  # For equality tests
+
```

Comment:
We don't need to match pygaqme exactly, but we do want to make the public API we expose explicit, so it's clear when people are poking at something internal.

Using ```_VECTOR_EPSILON``` is fine, since it's then clear that it's internal. I would personally also mark classes like ElementwiseVector2Proxy, which are not intended to be used outside of math.py as such, even though that difffers from what pygame itself exposes.